app/src/main/java/com/simplecity/amp_library/services/EqualizerService.java

```diff
@@ -246,11 +246,7 @@ public void onDestroy() {
 
         unregisterReceiver(mAudioSessionReceiver);
 
-        for (EffectSet gone : mAudioSessions.values()) {
-            if (gone != null) {
-                gone.release();
-            }
-        }
+        mAudioSessions.values().stream().filter(s -> s != null).forEach(EffectSet::release);
```

Comment:
Can we give `s` a more appropriate name here? I can't tell what it's supposed to be.